code/modules/mob/living/simple_animal/hostile/terror_spiders/verbs.dm

```diff
@@ -0,0 +1,160 @@
+
+/mob/living/simple_animal/hostile/poison/terror_spider/verb/ShowGuide()
+	set name = "Show Guide"
+	set category = "Spider"
+	set desc = "Learn how to spider."
+	to_chat(src, "------------------------")
+	to_chat(src, "Intro:")
+	to_chat(src, "- Terror Spiders are a bioweapon, created when the Syndicate mixed Giant Spider and Xenomorph DNA.")
+	to_chat(src, "- Ruled by Queens, they are aggressive, and very good in melee combat.")
+	to_chat(src, " ")
+	to_chat(src, "Communications:")
+	to_chat(src, "<B>- You speak over the Terror Spider hivemind by default. All other TS hear this. To speak common, use :9 or .9 </B>")
+	to_chat(src, "- Terror Spiders are the nuke ops of spiders. They work as a team. Communicate regularly!")
+	to_chat(src, " ")
+	to_chat(src, "Verbs:")
+	to_chat(src, " - Show Guide - Shows this guide.")
+	to_chat(src, " - Web - Spins a terror web. Non-spiders get trapped if they touch a web.")
+	to_chat(src, " - Eat Corpse - Eat the corpse of a dead foe to boost your regeneration")
+	to_chat(src, "------------------------")
+	to_chat(src, " ")
```

Comment:
For consistency's sake, might wanna do `list.join` and `to_chat` like you do with the other spiders